app/ont/ont.rb

```diff
@@ -6,6 +6,10 @@ def self.table_name_prefix
     'ont_'
   end
 
+  def self.library_attributes
+    []
```

Comment:
I believe this would be `volume, concentration, insert_size, tag_id`. Might be better just throwing an error if this method is reached as unsupported. e.g. `raise StandardError, 'Unsupported'`